clients/src/main/java/org/apache/kafka/common/requests/FindCoordinatorResponse.java

```diff
@@ -45,12 +43,12 @@
     private final Errors error;
     private final Node node;
 
-    public GroupCoordinatorResponse(Errors error, Node node) {
+    public FindCoordinatorResponse(Errors error, Node node) {
```

Comment:
Can we add an error message to go along with the error code? It already happens today that a `GroupCoordinatorNotAvailable` on the client is generally not enough to know what's wrong. We have to check the broker logs and hope that additional information is available there (and we had to add additional logging, after the fact, in some cases). Looks like this is about to become a bigger issue with this change.

CreateTopicsResponse is an example of a protocol response that includes an error message along with the error code.